net/wireguard-go/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                wireguard-go
+version             0.0.20180613
+categories          net
+platforms           darwin
+license             GPL-2
+maintainers         {isi.edu:calvin @cardi} openmaintainer
+description         Userspace Go implementation of WireGuard VPN
+long_description    \
+    WireGuard-go is the userspace Go implementation of WireGuard, a \
+    novel VPN that runs inside the Linux Kernel and utilizes \
+    state-of-the-art cryptography. It aims to be faster, simpler, \
+    leaner, and more useful than IPSec, while avoiding the massive \
+    headache. It intends to be considerably more performant than \
+    OpenVPN. WireGuard is designed as a general purpose VPN for \
+    running on embedded interfaces and super computers alike, fit for \
+    many different circumstances. It runs over UDP.
+
+homepage            https://www.wireguard.com/
+master_sites        https://git.zx2c4.com/wireguard-go/snapshot/
+use_xz              yes
+
+checksums           rmd160  eaca7df0a711c0f02f288c41bed6b0375f827773 \
+                    sha256  3e22e6f2a715f05f9bbc5b1a9c737ab2edc8f26b2af61f9cc31f83391cd663ff \
+                    size    53628
+
+depends_build       port:dep \
+                    port:git \
+                    port:go
+
+use_configure       no
+
+build.target        {}
```

Comment:
I can't remember if the `{}` is actually necessary or if just saying `build.target` is sufficient. You might check.